packages/voicechat-server/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "voicechat-server",
```

Comment:
Oh wait, so this needs a server to be host on? cc @yisiliu

Reply:
Yes, exactly. But this is just a signaling server and no voice is transferred across this server. This has the massive advantage over cloud / SaaS providers in that space in terms of costs.

You can easily deploy this server on a small 5$ Digital-Ocean droplet for example using something like pm2.

Reply:
i need this server deploy guide.

If you, can provide `Dockerfile` better (We will publish to https://hub.docker.com/u/dimensiondev)